pkg/ccl/streamingccl/streamingest/stream_replication_e2e_test.go

```diff
@@ -504,9 +504,9 @@ func TestTenantStreamingPauseResumeIngestion(t *testing.T) {
 	srcTime = c.srcCluster.Server(0).Clock().Now()
 	c.waitUntilHighWatermark(srcTime, jobspb.JobID(ingestionJobID))
 	c.compareResult("SELECT * FROM d.t2")
-	// Confirm this new run resumed from the previous checkpoint.
-	require.Equal(t, pausedCheckpoint,
-		streamIngestionStats(t, c.destSysSQL, ingestionJobID).IngestionProgress.StartTime)
+	//// Confirm this new run resumed from the previous checkpoint.
+	//require.Equal(t, pausedCheckpoint,
+	//	streamIngestionStats(t, c.destSysSQL, ingestionJobID).IngestionProgress.StartTime)
```

Comment:
We deleted this field from `IngestionProgress` since it was not really used for anything, and was one more timestamp to keep track of. I pushed a new test if you want to take a look!

Reply:
no no, that's fine as long as we don't have commented code, thanks!

Reply:
yup, the commented code was my bad. Thanks for pushing for me to right a test 😋